src/tests/Microsoft.PowerFx.Core.Tests/DisplayNameTests.cs

```diff
@@ -98,6 +98,18 @@ public void ValidateDisplayNames(string inputExpression, string outputExpression
                 var outInvariantExpression = _engine.GetInvariantExpression(inputExpression, r1);
                 Assert.Equal(outputExpression, outInvariantExpression);
             }
```

Comment:
nit: we can share lines 91..100 across both recordtypes. 

line 102:
var r2 = new LazyRecordType();

foreach(var r in [r1, r2]) {
  // lines 91...100
}

Reply:
Updated